packages/mobile/src/screens/profile-screen/EmptyProfileTile.tsx

```diff
@@ -0,0 +1,38 @@
+import { User } from 'audius-client/src/common/models/User'
+import { getAccountUser } from 'audius-client/src/common/store/account/selectors'
+import { Nullable } from 'audius-client/src/common/utils/typeUtils'
+
+import { EmptyTile } from 'app/components/core'
+import { useSelectorWeb } from 'app/hooks/useSelectorWeb'
+
+const messages = {
+  you: 'You',
+  haveNot: "haven't",
+  hasNot: "hasn't"
+}
+
+export const useEmptyProfileText = (
+  profile: Nullable<User>,
+  baseMessage: string
+) => {
+  const accountUser = useSelectorWeb(getAccountUser)
+
+  if (!profile) return ''
+  const { user_id, name } = profile
+  const isOwner = user_id === accountUser?.user_id
+  const youAction = `${messages.you} ${messages.haveNot}`
```

Comment:
Wouldn't this cause you to have a message like 'You haven't haven't created any albums' if the emptyText passed in is 'haven't created any albums yet'?

Reply:
yeah totally whoops, fixin